libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts

```diff
@@ -4,9 +4,19 @@ import { CommonModule } from '@angular/common';
 import { ConsultancyRoutingModule } from './consultancy.routing';
 
 import { ConsultancyPageComponent } from './main/consultancy-page/consultancy-page.component';
+import { ConsultancyHeroSectionComponent } from './consultancy-hero-section/consultancy-hero-section.component';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
+import { AppHeaderModule } from '@elewa-website/elements/layout/header';
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
 
```

Comment:
Check the order of imports; elements imports should appear before local imports

Reply:
@AnnaL001 *resolved

Thanks again!